express/parsers/apps/espresso/formats/espresso_640xml.py

```diff
@@ -0,0 +1,141 @@
+from typing import Sequence, List, Any, Optional, Dict, Union
+
+import xml.etree.ElementTree as ET
+from express.parsers.formats.xml import BaseXMLParser
+from express.parsers.settings import Constant
+
+
+def traverse_xml(node: ET.Element, *pathway: Sequence[str]) -> ET.Element:
+    """
+    Goes to a node in the node's path. For example, if we have a node tree that looks like A->B->C->D, then
+    we could call go_to_node(B, ["C", "D"]) to return a reference to node D. Mostly this is useful to avoid numerous
+    calls to "node.find('some_tag').find('some_other_tag').find('yet-another-tag')".
+    """
+    if isinstance(pathway, str):
+        pathway = (pathway,)
+    for step in pathway:
+        node = node.find(step)
+    return node
+
+
+def string_to_vec(string: str, dtype: type = float, sep: Optional[str] = None) -> List[Any]:
+    """
+    Given a string and some delimiter, will create a vector with the specified type.
+
+    Args:
+        string (str): The string to convert, for example "6.022e23 2.718 3.14159"
+        dtype (type): The type to convert into. Must support conversion from a string. Defaults to `float`
+        sep (Optional[str]): Delimiter for the the string. Defaults to whitespace.
+    Returns:
+        List[Any]: A list that has the correct type, for example [6.022e23, 2.718, 3.14159]
+    """
+    result = [dtype(component) for component in string.split(sep)]
+    return result
+
+
+def find_tag(node: ET.Element, tag: str):
+    """
+    DFS for tag in the node tree. Be aware that this is pretty imprecise, and just finds the *first* one it sees.
+    """
+    if node.tag == tag:
+        return node
+
+    for child in node:
+        result = find_tag(child, tag)
+        if result is not None:
+            return result
```

Comment:
The above functions should be in a central place - perhaps utils for XML or smth like that

Reply:
Sure, I can move them somewhere - I'll make them part of the parent XML class since they go with the XML functionality. We don't use this DFS function anywhere (turns out I didn't actually need it since they standardize the XML); since it's so simple, I'll get rid of it and keep the others.

Reply:
Ended up moving the `traverse_xml` method to the parent XML class. Kept the `string_to_vec` function inside the new Espresso parser, since that method is specific to the way Espresso reports vectors